File: Python/7py 1/closest_multiple_10.py
"""
|--------------------------------------------------------------------------
| Return the closest number multiple of 10
|--------------------------------------------------------------------------
|
"""

def closest_multiple_10(i):
   return round(i / 10) * 10


def closest_multiple_10(i):
   return round(i, -1)


def closest_multiple_10(i):
   return int(i/10+.5)*10


print(closest_multiple_10(54), 50)
print(closest_multiple_10(55), 60)

File: PHP/7php 1/closest_multiple_10.php
<?php
/*
|--------------------------------------------------------------------------
| Return the closest number multiple of 10
|--------------------------------------------------------------------------
|
*/

function closest_multiple_10($n) {
   return round($n / 10) * 10;
}


function closest_multiple_10($n) {
   return round($n, -1);
}


function closest_multiple_10($n) {
   return round(($n/10),0,PHP_ROUND_HALF_UP)*10;
}


print_r(closest_multiple_10(54) . ' ' . 50 . '
');
print_r(closest_multiple_10(55) . ' ' . 60 . '
');